samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
Actually this file can be found in Khronos repos on GitHub, I believe that true origin is there.